geometry_msgs/PoseStamped frontierLocation[5]